status/data_deliveries_plot.py

```diff
@@ -0,0 +1,58 @@
+import json
+import datetime
+from status.util import dthandler, SafeHandler
+
+
+class DataDeliveryHandler(SafeHandler):
+    """ Handles the api call to proj_staged data
+    Loaded through /api/v1/proj_staged/([^/]*)$
+    """
+    def get(self, search_string=None):
+        staged_files_view = self.application.projects_db.view("project/staged_files_irma")
+        summary_view = self.application.projects_db.view("project/summary")
+        delivered_data = {}
+        if not search_string:
+            last_month = datetime.datetime.now()-datetime.timedelta(days=30)
+            first_term = last_month.isoformat()[:10]
+            second_term = datetime.datetime.now().isoformat()[:10]
+            search_string = "{}--{}".format(first_term, second_term)
+        searches = search_string.split('--')
```

Comment:
You don't need this right? You can just use `first_term` and `second_term` further down where you use `searches[0]` and `searches[1]` now. You could get rid of the line above then as well.

Reply:
I realised I made a mistake. Please ignore.